BackofficeBundle/Resources/public/coffee/orchestraLib.coffee

```diff
@@ -165,6 +165,7 @@ activateHidden = (hidden) ->
 
 #ACTIVATE FORM JS
 activateForm = (view, form) ->
+  runAllForms()
```

Comment:
there are not conflict with other activation like select2, because in runAllForms they are an activation of select2 which is diffrent from our ?